src/libcollections/fmt.rs

```diff
@@ -366,11 +366,10 @@
 //! These can all be interpreted as flags for a particular formatter.
 //!
 //! * `+` - This is intended for numeric types and indicates that the sign
-//!         should always be printed. Positive signs are never printed by
-//!         default, and the negative sign is only printed by default for the
-//!         `Signed` trait. This flag indicates that the correct sign (`+` or `-`)
-//!         should always be printed.
-//! * `-` - Currently not used
+//!         (`+` or `-`) should always be printed. Positive signs are never
+//!         printed by default.  This behavior is controlled by using the
+//!         [pad_integral](struct.Formatter.html#method.pad_integral) method.
```

Comment:
Url shouldn't be formatted this way. First the `[pad_integrad]` then below `[pad_integral]: struct.Formatter.html#method.pad_integral`